app/models/subnet.rb

```diff
@@ -187,7 +187,14 @@ def self.import(proxy)
     ProxyAPI::DHCP.new(:url => proxy.url).subnets.map do |s|
       # do not import existing networks.
       attrs = { :network => s["network"], :mask => s["netmask"] }
+
       next if first(:conditions => attrs)
+      if s["options"]
+        attrs.merge!({:gateway => s["options"]["gateway"],
```

Comment:
There's a PR in smart-proxy that introduces support for this for the isc provider, but not the MS windows one. We either need: add similar functionality on Windows, or be able to handle response that doesn't contain these newly introduced fields. 

Could you add a test for this?